src/Accounts.js

```diff
@@ -0,0 +1,517 @@
+/* eslint-disable max-len */
+
+import { each, keys, includes, trim, isEmpty, merge, extend } from 'lodash';
+import { Iterable } from 'immutable';
```

Comment:
You could use Immutable throughout your redux implementation. Some of the reducer cases (below) could be written more tersely. Are you not doing so, in order to keep this package more generic for consumers not using immutable?